### DISCUSSION TITLE
my account is blocked

### DISCUSSION THREAD

**Discussion by JoaoYamaguti** [vercel/community#3237] [Help] [answered]
### Summary

Today i try to login in Vercel dashboard for first time, but appers this error message: "This user account is blocked."
And the Vercel app is already included in my Github profile.

![vercelErro](https://github.com/vercel/community/assets/105678854/38bec5d6-90c5-4d78-94a6-a5095dbff5f8)

### Steps to Reproduce

Try to Login using Github;

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.

**Reply by okadajunji** [reply]
Excuse me, I had the same problem, so I sent an email, but I haven't received a response even after a month.

**Reply by BleuWaters** [reply]
@amyegan Please, I have not got any response from registration@vercel.com. What do I do next. Thanks

**Reply by pintoderian** [reply]
Please, like the users above I have the same problem, more than a month and I have not had a response.

**Reply by alexander0205** [reply]
@JoaoYamaguti @pintoderian @okadajunji  did you get any answer ?

**Reply by alexander0205** [reply]
@amyegan  I send the email but, I haven't get any solution
